custom-namespaces.yaml

```diff
@@ -660,9 +660,7 @@ ads:
 firefox_ios:
   pretty_name: Firefox iOS
   owners:
-    - athomas@mozilla.com
     - darcese@mozilla.com
-    - kik@mozilla.com
```

Comment:
Could you provide some context on why I'm being removed from this list?

Reply:
Update pushed -- I'll dm you via slack